azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -61,7 +60,7 @@
     void generalize() throws CloudException, IOException;
 
     /**
-     * power off (stop) the virtual machine.
+     * Power off (stop) the virtual machine.
      * <p>
      * You will be billed for the compute resources that this Virtual Machine uses
```

Comment:
Since this is a full sentence (and it is part of the top description, not a @param or @return), it'd be good to consistently end those with a "."